recipes/bamhash/1.1/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+export LDFLAGS=-L$PREFIX/lib
+export CXXFLAGS=-I$PREFIX/include
+
+make all
+
+mv bamhash_checksum_bam $PREFIX/bin
+mv bamhash_checksum_fastq $PREFIX/bin
```

Comment:
Same problem with version 1.0. I downloaded it, and tried  `bamhash_checksum_fasta --version` and I get `bash: bamhash_checksum_fasta: command not found`.